config/sync/core.entity_form_display.paragraph.accordion.default.yml

```diff
@@ -3,6 +3,7 @@ langcode: en
 status: true
 dependencies:
   config:
+    - field.field.paragraph.accordion.field_p_a_accordion_heading
     - field.field.paragraph.accordion.field_p_a_accordion_item
```

Comment:
It's up to you if you wanna change this or not but this field should be called 'accordion_items' since the field references a paragraph which can multiple items.

Reply:
I understand, but I'm okay with the field name. I think it's fine.

Reply:
Just wait till i wake and look at it tomorrow loool

Reply:
Changing it as we speak actually lmaooo